src/library/baseplaylistfeature.cpp

```diff
@@ -478,6 +484,20 @@ void BasePlaylistFeature::slotExportPlaylist() {
     }
 }
 
+void BasePlaylistFeature::slotExportTrackFiles() {
+    int rows = m_pPlaylistTableModel->rowCount();
```

Comment:
Both PlaylistFeature and SetlogFeature use a PlaylistTableModel so you don't have to worry about that -- se slotExportPlaylist above which does the same sort of thing.